.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Run Test
         run: make test
       - name: Upload coverage artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
I believe this is already fixed by mas @irainia
Please do git rebase mas @Mayurjag